lib/sql.js

```diff
@@ -610,6 +631,23 @@ SQLConnector.prototype.buildUpdate = function(model, where, data, options) {
 };
 
 /**
+ * Build the UPDATE statement for replacing
+ * @param {String} model The model name
+ * @param {Object} where The where object
+ * @param {Object} data The data to be changed
+ * @param {Object} options The options object
+ * @param {Function} cb The callback function
+ * @returns {ParameterizedSQL} The UPDATE SQL statement for replacing fields
+ */
+SQLConnector.prototype.buildReplace = function(model, where, data, options) {
```

Comment:
I think this is an internal/private method that should not be called by connector users. If that's correct, then the method name should be prefixed with underscore, i.e. `_buildReplace`.